README.md

```diff
@@ -44,9 +44,11 @@ histogram in the file named `"histogram.svg"`.
 
 ## Limitations
 
-- Libfunc frequency results haven't been checked yet.
-- Log of transaction traces to JSON requires a lot of memory allocation. It
-  still needs to be optimised.
+Libfunc frequency results haven't been checked yet.
+
+## Requirements
+
+This crate is compatible with Rust 1.78. Both x86 and ARM are supported.
```

Comment:
Why specifically this version?

Reply:
Blockifier is not compatible with 1.81 and above. After moving to sequencer, it will be possible to use more recent versions of Rust. However `pathfinder_simd` is not compatible with 1.83 on ARM.

Reply:
Okay, so please explain that here in the readme.